### DISCUSSION TITLE
Eve chart - homeSpan temp sensor

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [HomeSpan/HomeSpan#180] [Q&A]
Is there any possibility for the eve application to display data on a graph?

my temperature sensor only shows the current temperature - it would be nice to see a graph

as in the picture from this link, you can see that maybe it is possible?

https://www.evehome.com/en/eve-weather

if not via eve then maybe it could be done in a different way?

**Comment by HomeSpan** [comment]
The graphing feature is part of the Eve App but is not available in the Home App.  Eve uses some proprietary (undocumented) Characteristics that can in principal be added to HomeSpan, but may require more work.  Please see Issue #169.

**Reply by aLAN-LDZ** [reply]
Can I integrate this with homespan to make www with charts `https://randomnerdtutorials.com/visualize-esp32-esp8266-sensor-readings-from-anywhere/` ?

**Comment by aLAN-LDZ** [comment]
I understand, it's a pity - such an option would be useful. I'm not that good at programming to write it myself. But maybe in future updates you will think about such a feature?

You did a great job anyway, thank you!
